src/api/telemetry/TelemetryAPI.js

```diff
@@ -529,7 +529,7 @@ define([
         }
 
         this.openmct.notifications.error(message);
-        console.error(detailMessage);
+        console.warn(detailMessage);
```

Comment:
is there a scenario where this should still throw an error instead of a warn?

Reply:
Hmm I'm not sure... @jvigliotta any incite? It does send an error notification as well.

Reply:
@alizenguyen let's add an e2e test for this

Reply:
I think the error notification is sufficient. I think there are many times where a provider doesn't exist for an object, I don't want to throw console errors for that each time, I think a warning would suffice. I mean it's evident from the view as well...